block/src/Pos/Block/Logic/VAR.hs

```diff
@@ -28,22 +29,24 @@ import           Pos.Block.Logic.Internal (BypassSecurityCheck (..), MonadBlockA
                                            applyBlocksUnsafe, normalizeMempool,
                                            rollbackBlocksUnsafe, toSscBlock, toTxpBlock,
                                            toUpdateBlock)
-import           Pos.Block.Slog (ShouldCallBListener (..), mustDataBeKnown, slogVerifyBlocks)
+import           Pos.Block.Slog (ShouldCallBListener (..), mustDataBeKnown)
 import           Pos.Block.Types (Blund, Undo (..))
 import           Pos.Core (Block, HeaderHash, epochIndexL, headerHashG, prevBlockL)
 import qualified Pos.DB.GState.Common as GS (getTip)
-import           Pos.Delegation.Logic (dlgVerifyBlocks)
 import           Pos.Lrc.Worker (LrcModeFull, lrcSingleShot)
-import           Pos.Ssc (sscVerifyBlocks)
-import           Pos.Txp.Settings (TxpGlobalSettings (..))
 import qualified Pos.Update.DB as GS (getAdoptedBV)
-import           Pos.Update.Logic (usVerifyBlocks)
 import           Pos.Update.Poll (PollModifier)
 import           Pos.Util (neZipWith4, spanSafe, _neHead)
 import           Pos.Util.Chrono (NE, NewestFirst (..), OldestFirst (..), toNewestFirst,
                                   toOldestFirst)
 import           Pos.Util.Util (HasLens (..))
 
+import           Pos.Block.Slog.Logic (slogVerifyBlocks)
```

Comment:
I personally do not like this, but that's subjective.

Reply:
Let's see what @pva701 thinks.

Reply:
don't like :(

Reply:
Reverted.